### DISCUSSION TITLE
Solving Scratch SVG bug

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7084] [Ideas]
I noticed a project in the Scratch website (two years old...) that shows an SVG bug in Scratch, https://scratch.mit.edu/projects/543603618/. I tried opening the project with the fix-uploaded-svgs addon enabled (that I don't know exactly what it solves) and the bug is still there.

Indeed, while the image on the stage is fine, when I go to the costume editor the labels are off, and the greenflag and white arrow at the end of the forever block are both missing. If I change the image in the editor, then the new, wrong image is shown on the stage.

**Comment by CST1229** [comment]
By the way, the SVG uploads fix addon currently only fixes off-center/scale images caused by the width/height properties being 100%, as shown in its [addon image](https://scratchaddons.com/addons/#addon-fix-uploaded-svgs):
![image](https://github.com/ScratchAddons/ScratchAddons/assets/68464103/f18902ef-a724-493e-b9e0-9aead1fc9b23)

**Reply by WorldLanguages** [reply]
Yeah, the addon fixes the bug *while the file is being uploaded* and as far as I know it was useful for SVGs imported from Affinity Designer some years ago.

**Comment by BroJac5246** [comment]
Has this been directly reported to the Scratch Team yet?

**Reply by sfederici** [reply]
I have no idea. But usually the Scratch team doesn't invest much on solving this kind of problems. I myself reported bug to the team that haven't been fixed in a year or more

**Reply by sfederici** [reply]
I did it now

**Reply by WorldLanguages** [reply]
It would be nice if this had a post in the bugs&glitches Scratch forums

**Reply by sfederici** [reply]
Yes, this is exactly what I did. I created a post in the bugs&glitches Scratch forums https://scratch.mit.edu/discuss/topic/735991/
